_data/simple-icons.json

```diff
@@ -6929,6 +6929,11 @@
             "hex": "8D6748",
             "source": "https://mochajs.org/"
         },
+        {
+            "title": "Modrinth",
+            "hex": "1BD96A",
+            "source": "https://modrinth.com/"
```

Comment:
Please don't pull directly from the site. Use the art repository: https://github.com/modrinth/art

Reply:
As per documentation `Official high-quality brand logos and brand colors can usually be found in the following locations: Website headers (you can use SVG-grabber for Chrome)`
I don't think the choice really matters since both are actual brand logos with the correct logo specifications.

Reply:
The art repository is more up to date and has proper vectors. The vectors in the art repository are also more likely to have higher quality than is found elsewhere on the site (the assets in the art repository were exported more recently than any of the logos that are currently in use). The production site is using an outdated favicon, for example, as outlined in my other comment.

Reply:
It is always better to use icons from official brand guidelines package or repository when available! Dont forget to use the permanent Github link as stated in [our contributing guidelines for Github source](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines).

While at it, you could add https://github.com/modrinth/art as guidelines and add the licence entry as well in `simple-icons.json`

Reply:
As you can see, there are numerous paths in the favicon, and changing one disrupts the entire icon. I tried clipping, but it didn't work while optimizing and combining paths.
So it's up to you whether or not to merge the previous modification, as I won't be working on this favicon.

![Screenshot (3808)](https://user-images.githubusercontent.com/74038190/187250629-0004ed13-00d5-4bad-91bf-8fdb3685892c.png)
![Screenshot (3807)](https://user-images.githubusercontent.com/74038190/187250636-d4c873f0-18bd-49d6-a988-4ddd3a8ab2b1.png)

Reply:
@Anmol-Baranwal You will have to merge path together to have a single path. You can take a look at https://github.com/simple-icons/simple-icons/discussions/5076 to get help doing it depending on the software you are using.